Sources/JSONService.swift

```diff
@@ -0,0 +1,70 @@
+
+/// Default implementation of a JSON service.
+/// Serves your `Order` to a server and parses the respons.
+/// Response is delivered to you as a `JSONResult`.
+public class JSONService : Service {
```

Comment:
❓Are we going to subclass the Service? I thought the plan was to add an adapter or handler (what's in a name) to the configuration and than call this from the `Service` layer. Are am I mistaken?